build-system/compile/closure-compile.js

```diff
@@ -144,7 +147,21 @@ function gulpClosureCompile(compilerOptions, nailgunPort) {
     }
   }
 
-  return closureCompiler.gulp(initOptions)(compilerOptions, pluginOptions);
+  // Normalize the file path before pushing into Closure.  Closure don't follow
```

Comment:
Nit: Normalize the file path before pushing into Closure. Closure doesn't follow